pallets/scoring/Cargo.toml

```diff
@@ -0,0 +1,82 @@
+[package]
+name = "pallet-scoring"
```

Comment:
I have a fundamental question. Should we name it with a noun? (ie, **pallet-score** or action driven name is fine? ie, what is present here **pallet-scoring**.

Reply:
Scoring is good i think